line-bot-servlet/src/main/java/com/linecorp/bot/servlet/LineBotCallbackRequestParser.java

```diff
@@ -55,13 +57,13 @@ public LineBotCallbackRequestParser(
      * @return Parsed result. If there's an error, this method sends response.
      * @throws LineBotCallbackException There's an error around signature.
      */
-    public CallbackRequest handle(HttpServletRequest req) throws LineBotCallbackException, IOException {
+    public Map.Entry<String, CallbackRequest> handle(HttpServletRequest req) throws LineBotCallbackException, IOException {
```

Comment:
IMO, creating channel secret -> LineBotCallbackRequestParser map is better than modifying this method signature.